f8a_worker/workers/repo_dependency_finder.py

```diff
@@ -0,0 +1,160 @@
+
+        # Traverse the Repo to EPVs that have CVE's and report them
+        gremlin_str += "g.V(repo).out('has_dependency').has('cve_ids').valueMap();"
+        payload = {"gremlin": gremlin_str}
+        try:
+            rawresp = requests.post(url=GREMLIN_SERVER_URL_REST, json=payload)
+            resp = rawresp.json()
+            if rawresp.status_code != 200:
+                raise TaskError("Error creating repository node for {repo_url} - "
+                                "{resp}".format(repo_url=github_repo, resp=resp))
+        except Exception:
+            self.log.error(traceback.format_exc())
+            raise TaskError(
+                "Error creating repository node for {repo_url}".format(repo_url=github_repo))
+            return None
+
+        report = self.generate_report(repo_cves=resp, deps_list=deps_list)
+        return report
+
+    def get_cve(self, github_repo, deps_list):
+        package_set = set()
+        version_set = set()
+        eco_set = set()
+        for epv in deps_list:
+            ecosystem = epv.split(':')[0]
+            eco_set.add(ecosystem)
+            version = epv.split(':')[-1]
+            version_set.add(version)
+            name = epv.replace(ecosystem + ':', '').replace(':' + version, '')
+            package_set.add(name)
+
+        gremlin_str = "g.V().has('pecosystem', within(eco_list)).has('pname', within(pkg_list))." \
+                      "has('version', within(ver_list)).in('has_dependency').dedup().as('rp')." \
+                      "out('has_dependency').has('cve_ids').as('epv').select('rp','epv')." \
+                      "by(valueMap());"
+        payload = {
+            'gremlin': gremlin_str,
+            'bindings': {
+                'eco_list': list(eco_set),
+                'pkg_list': list(package_set),
+                'ver_list': list(version_set)
+            }
+        }
+        try:
+            rawresp = requests.post(url=GREMLIN_SERVER_URL_REST, json=payload)
+            resp = rawresp.json()
+            if rawresp.status_code != 200:
+                raise RuntimeError("Error creating repository node for {repo_url} - {resp}"
+                                   .format(repo_url=github_repo, resp=resp))
+        except Exception:
+            self.log.error(traceback.format_exc())
+            raise RuntimeError("Error creating repository node for {repo_url}"
+                               .format(repo_url=github_repo))
+            return None
+
+        return resp
+
+    def generate_report(self, repo_cves, deps_list):
+        repo_list = []
+        for repo_cve in repo_cves.get('result').get('data', []):
+            epv = repo_cve.get('epv')
+            repo_url = repo_cve.get('rp').get('repo_url')[0]
+            name = epv.get('pname')[0]
+            version = epv.get('version')[0]
+            ecosystem = epv.get('pecosystem')[0]
+            str = ecosystem + ":" + name + ":" + version
```

Comment:
I would prefer if you renamed this variable instead of overwriting the system class `str`.